api/nodes/serializers.py

```diff
@@ -474,10 +474,10 @@ def get_registration_links_count(self, obj):
         return count
 
     def get_linked_by_nodes_count(self, obj):
-        return NodeRelation.objects.filter(child=obj, is_node_link=True).select_related('parent').exclude(parent__type='osf.collection').exclude(parent__type='osf.registration').count()
+        return NodeRelation.objects.filter(child=obj, is_node_link=True, parent__is_deleted=False).exclude(parent__type='osf.collection').exclude(parent__type='osf.registration').count()
```

Comment:
Would it make sense to do do `parent__type='osf.node'` instead of excluding non-nodes?